internal/main.go

```diff
@@ -33,8 +35,26 @@ func RunTests(opts RunTestsOpts) {
 	// Exclude by package prefix
 	testPkgs := allPkgs
 	if len(opts.Excludes) > 0 {
-		testPkgsStr := shCmd("grep", shArgs{"-Ev", getExcludePattern(opts.Excludes)}, allPkgsStr)
-		testPkgs = splitLines(testPkgsStr)
+		testPkgs = []string{}
+		excludeRegexs := make([]*regexp.Regexp, 0, len(opts.Excludes))
+		for _, exclude := range opts.Excludes {
+			regex, err := regexp.Compile("^" + exclude + "$")
```

Comment:
The existing code doesn't add `$` at the end. 
It is so that each entry is treated as a prefix while also allowing the user to add the `$` where needed.

Reply:
ok

Reply:
Donne